api/src/queries/project/project-view-queries.ts

```diff
@@ -3,6 +3,62 @@ import { getLogger } from '../../utils/logger';
 
 const defaultLog = getLogger('queries/project/project-view-queries');
 
+/**
+ * SQL query to get a single public (published) project.
+ *
+ * @param {number} projectId
+ * @returns {SQLStatement} sql query object
+ */
+export const getPublicProjectSQL = (projectId: number): SQLStatement | null => {
```

Comment:
I'm thinking it might be nice to separate the public queries from the non-public queries.

Maybe put them all under `api/src/queries/<project>/public` or alternatively `api/src/queries/public/<project>`

Reply:
This would also make them align with the paths, which are already under `api/paths/public`